test/integration/test-fixtures/image-go-bin-arch-coverage/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:latest as builder
+FROM golang:latest@sha256:cffaba795c36f07e372c7191b35ceaae114d74c31c3763d442982e3a4df3b39e as builder
```

Comment:
What does this do when `golang:latest` moves?

Reply:
That's a good question - I think this fails - let me pin this to a strict version